### ISSUE TITLE
TOPP workflow framework check in executable

### ISSUE THREAD

**Issue by Arslan-Siraj** [OpenMS/streamlit-template#51]

**Comment by Arslan-Siraj** [comment]
@axelwalter  i think we have to add other session variable exe to make difference between the local and exe?

**Comment by Arslan-Siraj** [comment]
@t0mdavid-m did you try about the threads/multiprocessing for windows executable?

**Comment by t0mdavid-m** [comment]
Yes it works now with new pipeline. I will create a PR tonight proposing other changes to the windows build pipeline.

**Comment by t0mdavid-m** [comment]
Was there anything else that was required for this issue? I think we have all the PRs now.
